### ISSUE TITLE
Uninstall plungins leave plugin folder

### ISSUE THREAD

**Issue by totorux** [OCSInventory-NG/OCSInventory-Server#54]
*It's hard to solve a problem when important details are missing, that why we added this template, to help you and us.*

### General informations
Operating system : Debian jessie

### Server informations
Perl version : v5.20.2
Mysql / Mariadb / Percona version : 5.5.54-0+deb8u1  

### OCS Inventory informations
Ocs server version : 2.3

### Problem's description
When installing and unstalling a plugin the plugin folder stay in /etc/ocsinventory-server/perl/Apache/Ocsinventory/Plugins/
So if you install again the same plugin you have a Err_05 : Residual directory found in perl dir ( by default -> /etc/ocsinventory-server/perl/Apache/Ocsinventory/Plugins/PLUGIN_NAME/ ) in OCS

Regards

**Comment by brianda78** [comment]
I am having the same problem.  I have done some troubleshooting and think I have it narrowed down to the function delete_plugin found in the following PHP file.
/usr/share/ocsinventory-reports/ocsreports/plugins/main_sections/ms_plugins/functions_delete.php
I looked through the code and did not see where it is attempting to remove the perl plugin directory, so I think it just needs to be added.  Not sure what the code would be though as I don't know what the variable is that contains the path to the perl plugins, or even if there is one.  Just as a simple test I added this line to the function which has a hard coded path to delete the Sharedfolder plugin dir and it works for that, so I am pretty sure this is the correct place.
rrmdir("/etc/ocsinventory-server/perl/Apache/Ocsinventory/Plugins/Sharedfolder");
Hopefully one of the developers can assist with this.
